kano_qt_apps/lib/src/app.cpp

```diff
@@ -33,6 +33,9 @@ App::App(std::string app_file_path, std::shared_ptr<App> fb):
         return;
     }
 
+    // FIXME: move this to qDebug()
+    std::cout << "kano_qt_apps parsing app file: " << app_file_path << std::endl;
```

Comment:
I would love to, but I think we have a build problem and I can't see any qDebug statements. I will roll them back once I get it fully implemented and tested.